src/SFA.DAS.ApprenticeFeedback.Domain/Models/Feedback/FeedbackAttributeStatus.cs

```diff
@@ -1,8 +1,8 @@
-﻿namespace SFA.DAS.ApprenticeFeedback.Domain.Models
+﻿namespace SFA.DAS.ApprenticeFeedback.Domain.Models.Feedback
 {
     public enum FeedbackAttributeStatus
     {
         Disagree = 0,
         Agree = 1
     }
-}
+}
```

Comment:
Possibly minor, but I think some text editors won't recognise a text file as such if it doesn't end with a newline, and there's no newline at the end of this one.

Reply:
have added a new line